packages/twitch-electron-auth-provider/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"extends": "../../tsconfig",
 	"compilerOptions": {
 		"rootDir": "src",
-		"outDir": "lib"
+		"outDir": "lib",
```

Comment:
This is invalid JSON. Even though TS tolerates this, I would prefer if it wasn't.

Reply:
Fixed